### DISCUSSION TITLE
Need help with configuring custom domain with AWS EC2 and Route 53

### DISCUSSION THREAD

**Discussion by fpiechowski** [appwrite/appwrite#3686] [Q&A]
What I did up to this point:

1. Performed a fresh install of Appwrite 0.15.3 with host `appwrite.nightout.party` and the same custom domain name (`appwrite.nightout.party`)
2. Implemented creating and getting sessions (Google OAuth2) in my Flutter app, tested on Android Emulator, it worked.
3. Due to some warning logs (unrelated thing) I downloaded Google Play services in SDK Manager, removed all my emulator devices, restarted Android Studio, created a new device with newly downloaded API version.
4. Then, on `getSession(sessionId: "current")` I got `AppwriteException: general_unauthorized_scope, User (role: guest) missing scope (account) (401)`, researched Custom Domains topic in Appwrite docs.
5. Added a custom domain in my Appwrite and changed DNS record for my custom domain in Route 53 from A type to CNAME type and changed value from IPv4 address to Public IPv4 DNS of the EC2 instance so that my config reflects instructions from the docs. Unfortunately Appwrite was unable to verify domain.
6. I thought it's because I used custom domain as my host and API endpoint so I did `docker compose down`, then `up` and reinstalled Appwrite using EC2 instance's DNS name as host and `appwrite.nightout.party` as custom name.
7. When containers went up, I logged to the Appwrite. It still had my project from before the reinstall of Appwrite, the API endpoint value has been changed and the custom domain was still there but it still couldn't be verified successfully.

Records for my domain (bought from other provider than AWS and delegated to Route 53) :

```
(autogenerated) nightout.party         (NS) ns-1007.awsdns-61.net. ns-163.awsdns-20.com. ns-1194.awsdns-21.org. ns-1811.awsdns-34.co.uk.
(autogenerated) nightout.party         (SOA) ns-1007.awsdns-61.net. awsdns-hostmaster.amazon.com. 1 7200 900 1209600 86400
nightout.party	                       (A) <landing page instance IPv4 address, not related to my problem here>
appwrite.nightout.party	               (CNAME) xxx.eu-west-1.compute.amazonaws.com (EC2 Public IPv4 DNS)
```

Current API Endpoint (in project's overview): http://xxx.eu-west-1.compute.amazonaws.com/v1 (was appwrite.nightout.com on the first try)
Current custom domains: appwrite.nightout.com

I'm sure I just lack knowledge and I configured it the wrong way, so could anybody explain how it should be done in my case so that Appwrite is able to verify my custom domain?

**Comment by stnguyen90** [comment]
@fpiechowski, if you only need 1 domain for Appwrite, you don't need to set up a custom domain in the Appwrite Console (that's for additional domains for your project). Your domain name should be set in the .env file. Please refer to the certificates [docs](https://appwrite.io/docs/certificates) for additional info. Feel free to share the logs if you're still having issues.

**Reply by stnguyen90** [reply]
@fpiechowski, if this is resolved, would you please mark my response as the answer? Otherwise, sharing the logs on [Discord](https://appwrite.io/discord/) would be the best next step.
